Lesson24-signup-form/assets/assets/ezgi/styles.css

```diff
@@ -0,0 +1,143 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  body {
+    font-family: Arial, sans-serif;
+    min-width:320px ;
+    height: 100vh;
```

Comment:
It could be that the content needs more space, so safer approach is to use `min-height: 100vh;`